protocol.md

```diff
@@ -1,147 +1,30 @@
+This means hiring bias is eliminated for these tasks, and the thresholds to getting work are reduced. To illustrate, the traditional solution requires companies to do some promotion to attract intelligence that is able to solve
+their tasks. Next, an employee has to apply. Then the employer needs to filter the applications to find a suitable candidate. If a match is made, the two can reach a work agreement. They then have to trust each other to live up to their promises and disputes can be settled through law and regulation. All these activities can be omitted when using the TruCol protocol for deterministically verifiable tasks, such algorithmic optimisations of heuristics of NP-hard problems and applications of artificial intelligence.
 
-## Extended Syntax
 
-These elements extend the basic syntax by adding additional features. Not all Markdown applications support these elements.
+## Advantages and disadvantages
 
-<table class="table table-bordered">
-  <thead class="thead-light">
-    <tr>
-      <th>Element</th>
-      <th>Markdown Syntax</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td><a href="/extended-syntax/#tables">Table</a></td>
-      <td><code>
-          | Syntax      | Description |<br>
-          | ----------- | ----------- |<br>
-          | Header      | Title       |<br>
-          | Paragraph   | Text        |
-      </code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#fenced-code-blocks">Fenced Code Block</a></td>
-      <td><code>```<br>
-      {<br>
-      &nbsp;&nbsp;"firstName": "John",<br>
-      &nbsp;&nbsp;"lastName": "Smith",<br>
-      &nbsp;&nbsp;"age": 25<br>
-      }<br>
-      ```
-      </code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#footnotes">Footnote</a></td>
-      <td><code>
-        Here's a sentence with a footnote. [^1]<br><br>
+By providing a protocol instead of a service, we empower the users financially whilst increasing their autonomy by removing the need to rely on an over-arching bounty platform that takes money from the work of users. Sharing a completely open and deterministic payout protocol removes bias from the hiring process. We believe the early-adopters on the employer side of the protocol can gain knowledge and experience that may give them a tactical advantage in the domain of requirement specification for AI engines like GPT-3. It can be difficult to identify which development processes in a company are suitable to offload into the TruCol protocol. It can be difficult to write air-tight test specifications in sponsor contracts. We intend to work as a consultancy to assist companies in successfully navigating these challenges to help them get the most out of the TruCol protocol.
```

Comment:
`It can be difficult to write air-tight` into: `It can be challenging to write air-tight`